### DISCUSSION TITLE
A little bug on environment wrapper

### DISCUSSION THREAD

**Discussion by zixuan417** [Toni-SM/skrl#70] [Q&A]
In the **Isaac Gym wrapper** class, the `render` method is inappropriately overridden by your wrapper and thus can not render successfully. For example, the render method of `IsaacGymPreview3Wrapper` is:
```python
 def render(self, *args, **kwargs) -> None:
        """Render the environment
        """
        pass
```
which overrides the `render()` in `VecTask`.

**Comment by Toni-SM** [comment]
Hi @403forbiddennn 

Many thanks for the feedback.
I didn't notice the change made in IsaacGymEnvs last July
I am going to fix in the `develop` branch... to include it in the next release
